AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java

```diff
@@ -0,0 +1,70 @@
+package com.applovin.enterprise.apps.demoapp;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+
+import com.adjust.sdk.Adjust;
+import com.adjust.sdk.AdjustConfig;
+import com.adjust.sdk.LogLevel;
+import com.applovin.sdk.AppLovinMediationProvider;
+import com.applovin.sdk.AppLovinSdk;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class GlobalApplication
+        extends Application
+{
+    @Override
+    public void onCreate()
+    {
+        super.onCreate();
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance( this ).setMediationProvider( AppLovinMediationProvider.MAX );
+        AppLovinSdk.getInstance( this ).initializeSdk( config -> {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+
+            // set up Adjust SDK
+            String appToken = "{YourAppToken}";
+            String environment = AdjustConfig.ENVIRONMENT_SANDBOX;
+            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), appToken, environment );
+            adjustConfig.setLogLevel( LogLevel.VERBOSE );
+            Adjust.onCreate( adjustConfig );
```

Comment:
Let's inline params and keep comment consistent with iOS, and no need for verbose log setting
```suggestion
            // Initialize Adjust SDK
            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), "{YourAppToken}", AdjustConfig.ENVIRONMENT_SANDBOX );
            Adjust.onCreate( adjustConfig );
```